android/titanium/libv8-services.mjs

```diff
@@ -129,7 +131,7 @@ async function generateSnapshot(v8SnapshotHeaderFilePath, rollupFileContent) {
 	}
 
 	// Generate 'V8Snapshots.h' from template
-	const template = await util.promisify(ejs.renderFile)('V8Snapshots.h.ejs', { blobs }, {});
+	const template = await promisify(ejs.renderFile)('V8Snapshots.h.ejs', { blobs }, {});
```

Comment:
Nit, we might be able to remove the promisify here, it appears `ejs.renderFile` can return a promise since [2.5.8](https://github.com/mde/ejs/blob/main/CHANGELOG.md#v258-2018-03-25)